src/tdastro/sources/sncomso_models.py

```diff
@@ -143,7 +143,7 @@ def _evaluate(self, times, wavelengths, graph_state=None, **kwargs):
         Returns
         -------
         flux_density : `numpy.ndarray`
-            A length T x N matrix of SED values.
+            A length T x N matrix of SED values (in nJy).
```

Comment:
sncosmo flux unit is not nJy (It's in ergs/s/cm^2/AA). I did the conversion when I used them for passband integration. We could move the conversion here, but I'm not sure.